src/gradualizer_prelude.erl

```diff
@@ -170,3 +170,50 @@
 -spec proplists:lookup_all(any(), list()) -> [tuple()].
 -spec proplists:append_values(any(), list()) -> list().
 -spec proplists:substitute_aliases([{any(), any()}], list()) -> list().
+
+%% qlc
+
+-type(cache() :: ets | list | no).
```

Comment:
I guess qlc uses this old style with the parens - we usually don't use it any more.

Reply:
Thanks! Fixed.